frontend/src/app/recipe/components/detail/recipe-detail.component.html

```diff
@@ -1,32 +1,52 @@
 @if (_recipe$ | async; as recipe) {
-  <h1>{{ recipe.name }}</h1>
+<div class="row mb-3">
+  <div class="col">
+    <h1>{{ recipe.name }}</h1>
+    @if (recipe.description) {
+      <p class="lead">{{ recipe.description | firstSentence }}</p>
```

Comment:
I think this is something that should happen during the parsing process. It's probably something specific to chefkoch and nothing general.
I also would check then if the description starts with the title and remove that as well.

Reply:
I you don't want to do this as part of this PR it would be fine with me to create another item/task for it and for now just display what ever content the description contains.

Reply:
I delete the firstSentence pipe